test/ttmlir/Silicon/TTMetal/simple_eltwise.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: ttmlir-opt --ttir-load-system-desc="path=%system_desc_path%" --ttir-to-ttmetal-backend-pipeline --ttmetal-serialize-to-binary="output=%t.ttm" %s | FileCheck %s
+// UNSUPPORTED: true
```

Comment:
Sure!

Reply:
I switched it to `XFAIL: true` and the tests are still failing, am I invoking this incorrectly? It seems as the CI parses it as a failed test regardless.

Reply:
You can invoking it correctly, but it fails because the testing is actually passing! That's brings into question why you are making it UNSUPPORTED or XFAIL in the first place as it is passing?